src/pyFAI/app/pilx.py

```diff
@@ -45,14 +45,21 @@
 def main():
     parser = argparse.ArgumentParser()
     parser.add_argument("filename")
+    parser.add_argument("-d", "--data", dest="data_path",
+                        help="inner path to the dataset with the Raw Data", default=None)
+    parser.add_argument("-p", "--nxprocess", dest="nxprocess_path",
+                        help="inner path to the Nexus process with the integrated Data", default=None)
+
```

Comment:
could you set the default value of -p to `/entry_0000/pyFAI` to change less the the behavior of the code ?

Reply:
ok